deploy/podman/README.md

```diff
@@ -1,24 +1,74 @@
-# Deploy with podman
+# Deploy with Podman
 
 In this scenario the service and associated components are deployed onto your
 local host in containers using [Podman](https://podman.io/).
 
-## Requirements
+## General Requirements
 
-Make sure you have [podman](https://podman.io) version 3.3+ installed. If you
+Make sure you have [Podman](https://podman.io) version 3.3+ installed. If you
 must use an older version of podman, reference the [previous documentation and
 procedure](https://github.com/openshift/assisted-service/tree/v2.0.11#deploy-without-a-kubernetes-cluster)
 to avoid a [podman bug](https://github.com/containers/podman/issues/9609).
 
-Grab [pod.yml](./pod.yml) and [configmap.yml](configmap.yml) from this
-directory. No need to clone the whole repo.
+## Requirements for an HTTP based deployment
+
+### Files
+
+No need to clone the whole repo. Use:
+* [pod.yml](./pod.yml)
+* [configmap.yml](configmap.yml)
+
+### Environment Variables
 
 Change `IMAGE_SERVICE_BASE_URL` and `SERVICE_BASE_URL` in
 [configmap.yml](./configmap.yml) to match the hostname or IP address of your
 host. For example if your IP address is 192.168.122.2, then the
 `SERVICE_BASE_URL` would be set to <http://192.168.122.2:8090>. Port 8090 is
 the assisted-service API that agents will connect to.
 
+## Requirements for an HTTPS based deployment
+
+* Assisted UI supports HTTPS starting from UI version [v2.10.0](https://github.com/openshift-assisted/assisted-ui/releases/tag/v2.10.0)
+
+### SSL Certificates
+
+:warning: This example uses self-signed certificates for simplicity, substitute in valid certificates if they are available. Note that self-signed certificates are **not recommended** for use in production.
```

Comment:
s/,/;/ to have a proper separator between independent clauses.

Reply:
> Note that self-signed certificates are **not recommended** for use in production.

Why not? This seems like a reasonable use case for them. The service will only run long enough to deploy a cluster, only one client is likely to connect, that client is probably connecting to `localhost` anyway, and even if not, that client already has the cert on-disk and could make their browser trust it (which might be logistically easier than obtaining a cert from a trusted authority).

Reply:
I fully agree with your comment.
I intended to emphasize that users should be mindful of using self-signed certificates in production, which is not a good practice, to my understanding. However, users' discretion may still lead to using self-signed certs, which can be okay for the goal of this deployment, as you mentioned.

I'm open to changing the wording here if you have any suggestions.

Reply:
I would remove this warning entirely. For this short-lived use case, the only advantage I can think of to using a cert signed by a known CA is the convenience that the user's browser will already trust it.

Reply:
Removed 👍